doc/Rules.md

```diff
@@ -1288,7 +1288,7 @@ Note: To allow specific HTML elements, use the 'allowed_elements' parameter.
 
 Rationale: Raw HTML is allowed in markdown, but this rule is included for
 those who want their documents to only include "pure" markdown, or for those
-who are rendering markdown documents in something other than HTML.
+who is rendering markdown documents in something other than HTML?
```

Comment:
"those" is plural, so the verb change is unwarranted. And why the question mark at the end?...